writing-visualising/dashboards_rshiny.qmd

```diff
@@ -189,7 +189,7 @@ You will need:
 * Approval from your DD
 * If the data underlying the dashboard is currently unpublished, you will need to create dummy data to use in GitHub until the data becomes published (see [dummy data guidance](#using-dummy-data) section).
 
-To set up a new app, send the above to [Statistics Development Team](mailto:statistics.development@education.gov.uk). If your code is not yet hosted in the dfe-analytical-services area you can request for the repository to be moved at the same time as sending approvals.
+To set up a new app, send the above to [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk). If your code is not yet hosted in the dfe-analytical-services area you can request for the repository to be moved at the same time as sending approvals.
```

Comment:
does `dfe-analytical-services` need a link or is that overkill?

Reply:
Added in next commit